utide/_ut_constants.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import (absolute_import, division, print_function)
+
+import os
+
+from .utilities import convert_unicode_arrays, loadbunch
+
+
+_base_dir = os.path.join(os.path.dirname(__file__), 'data')
+_ut_constants_fname = os.path.join(_base_dir, 'ut_constants.npz')
+
+# At least for now, use NaNs rather than masked arrays.
+ut_constants = loadbunch(_ut_constants_fname, masked=False)
+ut_constants = convert_unicode_arrays(ut_constants)
+
+constit_names = list(ut_constants.const.name)
+
+# Make a dictionary for index lookups.
+constit_index_dict = dict([(name, i) for (i, name) in
+                           enumerate(constit_names)])
```

Comment:
We could also make this a class, could we not? 

```python
class Constants:
   def __init__(self):
        # load all the variables
        # do all the stuff
```

Then we would just import the class, and use it with `Constants.constit_index_dict`, or something like that.

Maybe we don't like that though, just throwing out an idea.

Reply:
I don't think that using a class is appropriate here.  You would need to import it *and* create an instance; but that would have no advantage over the simpler structure in which the loading is done at package import time, and the constants are then accessed directly.